src/worker.h

```diff
@@ -109,7 +109,8 @@ class Worker {
   // get info about scheduler state
   void scheduler_info(ClientContext &context, SchedulerInfoRequest &request, SchedulerInfoReply &reply);
   // get task statuses from scheduler
-  void task_info(ClientContext &context, TaskInfoRequest &request, TaskInfoReply &reply);
+  void task_info(ClientContext &context, TaskInfoRequest &request, TaskInfoReply &reply); 
+  PyObject* is_ready(std::vector<ObjectID> &objectids);
   // export function to workers
```

Comment:
`select` in `worker.cc` should return a vector of indices, and they should be unpacked and turned into a `PyObject*` in `raylib.cc`. The idea is that `worker.cc` is completely independent of Python and could be used by workers in multiple languages.